### PR TITLE
PT-07: create a postman collection for api endpoints #12

### PR DESCRIPTION
**Pull request by ValBaliyska**
Adding tests for Post, Get, Put, Delete methods and also some negative scenarios.

### CODE REVIEW

File: PennyTracker.Api/Properties/launchSettings.json [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
     },
     "PennyTracker.Api": {
       "commandName": "Project",
-      "applicationUrl": "https://localhost:5001;http://localhost:5000",
+      "launchBrowser": true,
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
-      }
+      },
+      "applicationUrl": "https://localhost:5001;http://localhost:5000"
```
Comment (mitko100) [inline PennyTracker.Api/Properties/launchSettings.json]: Please, do not change these settings

File: PennyTracker.BlazorServer/Startup.cs [hunk 2]
```diff
@@ -29,7 +29,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddServerSideBlazor();
             services.AddHttpClient<IExpenseService, ExpenseService>(client =>
             {
-                client.BaseAddress = new Uri("https://localhost:44326/");
+                client.BaseAddress = new Uri("https://localhost:44361/");
```
Comment (mitko100) [inline PennyTracker.BlazorServer/Startup.cs]: Please do not change these settings

File: Postman.Tests/Penny Tracker API tests.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,300 @@
+{
```
Comment (mitko100) [inline Postman.Tests/Penny Tracker API tests.postman_collection.json]: Move the file into main directory, It's not intended to be in the test folder.

File: PennyTracker.BlazorServer/Startup.cs [hunk 4]
```diff
@@ -29,7 +29,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddServerSideBlazor();
             services.AddHttpClient<IExpenseService, ExpenseService>(client =>
             {
-                client.BaseAddress = new Uri("https://localhost:44361/");
+                client.BaseAddress = new Uri("https://localhost:44326/");
```
Comment (mitko100) [inline PennyTracker.BlazorServer/Startup.cs]: This one also shoudl be reverted

File: PennyTracker.Api/Properties/launchSettings.json [hunk 5]
```diff
@@ -18,6 +18,7 @@
     },
     "PennyTracker.Api": {
       "commandName": "Project",
+      "applicationUrl": "https://localhost:5001;http://localhost:5000",
       "launchBrowser": true,
```
Comment (mitko100) [inline PennyTracker.Api/Properties/launchSettings.json]: Basicly we don't want to launch a browser

File: PennyTracker.BlazorServer/Startup.cs (hunk 2 above)
Comment (ValBaliyska) [reply PennyTracker.BlazorServer/Startup.cs]: reverted to 44326

File: Postman.Tests/Penny Tracker API tests.postman_collection.json (hunk 3 above)
Comment (ValBaliyska) [reply Postman.Tests/Penny Tracker API tests.postman_collection.json]: Moved

File: PennyTracker.BlazorServer/Startup.cs (hunk 4 above)
Comment (ValBaliyska) [reply PennyTracker.BlazorServer/Startup.cs]: It should be 44326, you can see above comment

File: PennyTracker.Api/Properties/launchSettings.json (hunk 5 above)
Comment (ValBaliyska) [reply PennyTracker.Api/Properties/launchSettings.json]: ok, get rid of it

File: PennyTracker.Api/Properties/launchSettings.json [hunk 6]
```diff
@@ -1,16 +1,17 @@
...
-      "applicationUrl": "http://localhost:62538",
+      "applicationUrl": "http://localhost:62538/",
       "sslPort": 44326
     }
   },
+  "$schema": "http://json.schemastore.org/launchsettings.json",
   "profiles": {
     "IIS Express": {
       "commandName": "IISExpress",
+      "launchBrowser": true,
```
Comment (mitko100) [inline PennyTracker.Api/Properties/launchSettings.json]: Still?

File: PennyTracker.Api/Properties/launchSettings.json [hunk 7]
```diff
@@ -1,16 +1,17 @@
-﻿{
-  "$schema": "http://json.schemastore.org/launchsettings.json",
+{
   "iisSettings": {
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:62538",
+      "applicationUrl": "http://localhost:62538/",
```
Comment (mitko100) [inline PennyTracker.Api/Properties/launchSettings.json]: Remove the last slash '/'

File: PennyTracker.Api/Properties/launchSettings.json (hunk 6 above)
Comment (ValBaliyska) [reply PennyTracker.Api/Properties/launchSettings.json]: removed

File: PennyTracker.Api/Properties/launchSettings.json (hunk 7 above)
Comment (ValBaliyska) [reply PennyTracker.Api/Properties/launchSettings.json]: done

**Comment by mitko100** [comment]
Looks good, let's wait CI to finish and merge this.
Thank you for providing the collection this will boost development process.

**Comment by mitko100** [comment]
Closing due to wrong branches
